docs/installation/deployment/kubernetes.rst

```diff
@@ -0,0 +1,280 @@
+-----------------------------------------
+
+Check your operation system packages and make sure you have installed a recent
+enough Python version. We recommend using Python 3.7.
+
+Create a virtual environment
+----------------------------
+
+Virtual environments isolate dependencies between environments. It gives us
+close control over the exact required versions.
+
+Create a virtualenv:
+
+.. code-block:: shell
+
+    [user@host]$ python3.7 -m venv env
+
+This creates a virtualenv named ``env``. Next, activate the virtualenv. You
+need to do this every time you want to use the deployment tooling.
+
+.. code-block:: shell
+
+    [user@host]$ source env/bin/activate
+
+Install dependencies
+--------------------
+
+First, navigate to the correct directory. In the folder where you placed the
+copy of the repository, change into the ``deployment`` directory:
+
+.. code-block:: shell
+
+    (env) [user@host]$ cd /path/to/open-zaak/deployment/
+
+To install the required dependencies, we use the Python package manager ``pip``:
+
+.. code-block:: shell
+
+    (env) [user@host]$ pip install -r requirements.txt
+
+Roughly said, this installs Ansible and the modules to talk to the PostgreSQL
+database and Kubernetes API.
+
+Deploying (automated or manual)
+===============================
+
+Ansible has the concept of *playbooks* - a predefined set of tasks to execute,
+logically grouped.
+
+Open Zaak ships with two playbooks:
+
+* ``provision.yml``:
+
+    * finishes the configuration of your Kubernetes cluster (if needed)
+    * initializes the application databases
+
+* ``apps.yml``:
+
+    * installs Open Zaak
+    * installs Open Notificaties
+
+You can run the Ansible-playbooks as-is (with some configuration through
+variables), or use them an inspiration for manual deployment.
+
+Provisioning
+------------
+
+Below you find some guidance to modify the provisioning specifically to your
+needs.
+
+I already have an ingress-controller
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set the variable ``needs_ingress`` in ``provision.yml`` to ``no``. Otherwise,
+Traefik 2.0 is set up as Ingress controller.
+
+I have a ``ReadWriteMany`` storage solution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set the variable ``needs_nfs`` in ``provision.yml`` to ``no``. Otherwise,
+a NFS-server is deployed to use as ``ReadWriteMany`` solution.
+
+.. todo:: streamline nfs/RWX solution!
+
+Database configuration
+^^^^^^^^^^^^^^^^^^^^^^
+
+The playbook will set up the application database user(s) with the correct,
+minimal permissions and will set up the databases for the applications. To be
+able to do this, you need superuser access. See the
+``vars/db_credentials.example.yml`` file for the example configuration.
+
+Both Open Zaak and Open Notificaties require database configuration to be
+defined in the ``vars/openzaak.yml`` and ``vars/opennotificaties.yml``
+variable files:
+
+.. code-block:: yaml
+
+    openzaak_db_name: openzaak  # name of the database to create
```

Comment:
Should we convert this to be a connect url?

Reply:
No, it messes with the Ansible parameters and the Django envvars